chapters/annotations.tex

```diff
@@ -204,14 +205,34 @@ \subsubsection{Plot Curves}\label{plot-curves}
 Variable replacements, as described in \cref{variable-replacements}, can be
 used in the \lstinline!legend! of \lstinline!Curve!
 
+\subsubsection{Escape sequences}\label{text-markup-escape-sequences}
+
+In an attribute inside a figure where the variable replacements of \cref{variable-replacements} or the text markup of \cref{text-markup-in-captions} can be used, the following use of \firstuse{text markup escape sequences} applies.  These escape sequences are applied after the application of other markup, and is not applied at all inside some of the other markup, see details for the respective markup.
+
+The percent character `\%' shall be encoded \lstinline!%%!.  The following are all the recognized escape sequences:
+\begin{center}
+\begin{tabular}{c c l}
+\hline
+\tablehead{Sequence} & \tablehead{Encoded character} & \tablehead{Comment}\\
+\hline
+\hline
+\lstinline!%%! & `\%' & Only way to encode character. \\
+\lstinline!%]! & `]'  & Escape sequence only needed in link text. \\
+\hline
+\end{tabular}
+\end{center}
+
+\begin{nonnormative}
+With the percent character being encoded as \lstinline!%%!, the behavior of \lstinline!%! appearing in any other way than the escape sequences above, for variable replacement (\cref{variable-replacements}), or for link markup (\cref{text-markup-in-captions}) is undefined, and thus possible to define in the future without breaking backward compatibility.
+\end{nonnormative}
+
 \subsubsection{Variable Replacements}\label{variable-replacements}
-In most places where text for display is defined the final value of a result
-variable can be embedded by refering to the variable as
-\lstinline!%{inertia1.w}!.  This is similar to the \lstinline!Text! graphical primitive
-in \cref{text}.
+
+In the places listed in \cref{attributes-with-variable-replacements} where text for display is defined, the final value of a result variable can be embedded by refering to the variable as \lstinline!%{inertia1.w}!.  This is similar to the \lstinline!Text! graphical primitive in \cref{text}.
 
 \begin{table}[H]
 \caption{Attributes that can use variable replacements.}
+\label{attributes-with-variable-replacements}
```

Comment:
I normally try to use the exact text for sections. That would be attributes-that-can-use-variable-replacements not attributes-with-variable-replacements

Reply:
But this isn't even a title, it's a caption, and a caption could even consist of several paragraphs.  I mean, that I like short and stable labels probably doesn't come as a surprise, but while the scheme of systematic mapping of section titles to labels could sort of work (especially if one decides to never change a section title), systematic mapping of captions… really?